examples/service-clients/azure-client/external-controller/src/app.ts

```diff
@@ -66,6 +75,7 @@ const containerSchema = {
 		/* [id]: DataObject */
 		map1: SharedMap,
 		map2: SharedMap,
+		presence: ExperimentalPresenceManager,
```

Comment:
You are using single instance, but I'm just curious - what use-cases would be suitable for having multiple instances of PresenceManager ? Is it due to perf, usability, both?

Reply:
There aren't uses for multiple. There shouldn't be anything in container schema, but this is workaround to missing internal support - https://github.com/microsoft/FluidFramework/blob/main/packages/framework/presence/README.md#onboarding